package.json

```diff
@@ -2,8 +2,8 @@
   "name": "playroom",
   "version": "0.34.1",
   "description": "Design with code, powered by your own component library",
-  "main": "utils/index.js",
-  "types": "utils/index.d.ts",
+  "main": "utils/index.ts",
+  "types": "utils/index.ts",
```

Comment:
Do we want to ship typescript as source? The alternative would be a build step. We could just run `tsc` for now, and graduate to something that offers proper CJS/ESM support later.

Reply:
Reflecting on this, maybe for now we could just keep the separate `.js` and `.d.ts` files for simplicity, and think about a potential build process later on.